lib/apiservers/service/restapi/configure_vic_machine.go

```diff
@@ -47,9 +48,7 @@ func configureAPI(api *operations.VicMachineAPI) http.Handler {
 	})
 
 	// GET /container/version
-	api.GetVersionHandler = operations.GetVersionHandlerFunc(func(params operations.GetVersionParams) middleware.Responder {
-		return middleware.NotImplemented("operation .GetVersion has not yet been implemented")
-	})
```

Comment:
how about returning the version directly here without version_get.go (assuming it works)? 

```
api.GetVersionHandler = operations.GetVersionHandlerFunc(func(params operations.GetVersionParams) middleware.Responder {
        return operations.NewGetVersionOK().WithPayload(version.GetBuild().ShortVersion()) 
 })
```

Reply:
I believe that would work fine (and is similar to how I implemented it initially), but I think there's a lot of value in having the APIs handled consistently.

If `configure_vic_machine.go` is simply a dispatcher to logic in other files, and never the place where handler logic is implemented, I think it's easier to reason about. (It also allows for consistent patterns around, for example, testing.)